subjects/printfirsthalf/README.md

```diff
@@ -0,0 +1,46 @@
+## printfirsthalf
+
+### Instructions
+
+Write a function called `PrintFirstHalf()` that takes a `string` as an argument and returns printed the first half of a `string`.
+
+- If the length of the `string` is odd round it down.
```

Comment:
add comma: (...) the `string` is odd, round it down.

Reply:
It is fixed.